.gitignore

```diff
@@ -1,3 +1,5 @@
 .idea/*
 *.iml
 target/*
+src/main/java/mate/academy/spring/controller/TestInjectController.java
+src/main/java/mate/academy/spring/config/DataInitializer.java
```

Comment:
I don't quite understand why you added your DataInitializer class to .gitignore, is it written somewhere that we should do this?

Reply:
It's just a manual data injection classes for testing the app, so I think those shouldn't be in the project.

Reply:
@viprko  @V-Solianyk 
For this task is ok to push DataInitializer class.
If the classes are present in code base we should not add its to the .gitignore . 
If you change some file that shouldn't be pushed (e.g. TestInjectController.java), you need just manually exclude it from commit in intellij idea, or move this file to another change list in intellij idea. So your changes will not affect the other contributor's work.
The other contributors should not see mentions in .gitignore that you have only in your local project.